src/features/hotspots/settings/discovery/dedupeDiscoveryResponses.ts

```diff
@@ -0,0 +1,5 @@
+import { uniqBy } from 'lodash'
+import { DiscoveryResponse } from '../../../../store/discovery/discoveryTypes'
+
+export default (responses: DiscoveryResponse[]) =>
+  uniqBy(responses, (response) => response.hotspotAddress)
```

Comment:
i think you can also do this like `uniqBy(responses, 'hotspotAddress')`

Reply:
oh, cool. I've never used it like that before.

Reply:
For TS I think I prefer `uniqBy(responses, ({ hotspotAddress }) => hotspotAddress)`
That way we don't lose type safety.